package.json

```diff
@@ -14,10 +14,12 @@
   "dependencies": {
     "@discordjs/builders": "^1.3.0",
     "@discordjs/rest": "^1.3.0",
+    "card-deck": "^1.0.1",
     "discord-api-types": "^0.37.19",
     "discord.js": "^14.6.0",
     "firebase-admin": "^11.3.0",
     "fs": "^0.0.1-security",
+    "lodash": "^4.17.21",
```

Comment:
is this taking the entire lodash library or just the specific functions we are using?

Reply:
Probably the entire lodash library 👀